cmd/shortener/config.go

```diff
@@ -47,6 +64,24 @@ func (c *Configs) Parse() {
 		c.AddrDB = envAddrDB
 	}
 
+	// Проверка переменной окружения ENABLE_HTTPS
+	if envHTTPS := os.Getenv("ENABLE_HTTPS"); envHTTPS == "true" {
+		*c.HTTPS = true
+	}
+
+}
+
+// loadConfig загружает конфигурационный файл.
+func (c *Configs) loadConfig() error {
```

Comment:
Маленькое предложение по именованию - в нашем случае слово `Config` в названии метода немного избыточно, так как и так понятно, что метод относится к структуре `Configs`. При этом информации о том, что конфиг загружается из файла потеряна, а она является ключевой. Если переименуем метод `loadFromFile` решим обе проблемы. 
```suggestion
func (c *Configs) loadFromFile() error {
```

Reply:
спасибо за рекомендацию, применил